internal/formatters/results/timeline/formatter_test.go

```diff
@@ -24,14 +26,84 @@ func TestExxonFormatter_Order(t *testing.T) {
 		newResult("", 1500, 0, false),                    // Shouldn't show up with an empty transcript.
 	}
 
-	want := "500|0|One0\n500|1|One1\n500|0|One0.1\n750|1|Two\n7500|0|Three"
+	wantStr := `[
```

Comment:
Maybe declare this string/object at the end of the file, since it's so long?

Reply:
Also, I don't know why we are using a string representation and then marshaling it instead of just creating a regular go object.
If you are adding that step to test the marshal code, I would suggest adding that as a separate test function.

Reply:
Pure laziness--I had originally been comparing the json strings themselves but they kept flagged as different for stupid indenting reasons so in frustration I just marshaled them both.  Your nudge made me get un-lazy and convert to a go literal instead.